app/views/RoomsListView/index.js

```diff
@@ -416,19 +440,24 @@ export default class RoomsListView extends React.Component {
 					alert={item.alert}
 					unread={item.unread}
 					userMentions={item.userMentions}
+					isRead={!((item.archived !== true && item.open === true) && (item.unread > 0 || item.alert === true))}
```

Comment:
Create a method for that.
Split conditions and avoid double negation.
Add comments.
```jsx
get isRead() {
  ...
}

...

isRead={this.isRead}
```

Reply:
I cannot have a get as the item I am getting is coming from FlatList I have made a simple function ,